osu.Framework/Platform/Storage.cs

```diff
@@ -111,12 +111,14 @@ public virtual Storage GetStorageForDirectory(string path)
         /// Requests that a file be opened externally with an associated application, if available.
         /// </summary>
         /// <param name="filename">The relative path to the file which should be opened.</param>
-        public abstract void OpenFileExternally(string filename);
+        /// <returns>Whether the file was successfully opened.</returns>
+        public abstract bool OpenFileExternally(string filename);
 
         /// <summary>
         /// Opens a native file browser window to the root path of this storage.
         /// </summary>
-        public void PresentExternally() => OpenFileExternally(string.Empty);
+        /// <returns>Whether the file was successfully presented.</returns>
```

Comment:
`PresentExternally` presents the storage folder. Unsure if it should be called `storage` or `folder` or `path`.
```suggestion
        /// <returns>Whether the storage was successfully presented.</returns>
```